### DISCUSSION TITLE
Formatting JTE Files

### DISCUSSION THREAD

**Discussion by christophhagelkruys** [casid/jte#355] [Q&A]
Hi,
i would like to ask how you format your jte templates?
We tried to use the spotless maven plugin for our java code and jte files, but for the jte files it did not work.

**Comment by marcospereira** [comment]
My templates don't have Java/Kotlin code besides expressions in `@if`, `@for`, and such. They are usually pretty easy to keep well formatted, but I agree that it would be nice to integrate with a formatter.

> but for the jte files it did not work.

What do you mean here? Did it fail? Or it just didn't format the code?

**Comment by sgruendel** [comment]
see #535 , I've just released a Prettier plugin for that
